modules/social_features/social_core/config/install/image.style.social_xx_large.yml

```diff
@@ -1,13 +1,21 @@
 langcode: en
 status: true
-dependencies: {  }
+dependencies:
+  module:
+    - crop
```

Comment:
I see this as a dependency, but it's not added to the composer.json / drupal-org.make (nor in the social_core.install) maybe add this.. Seems to be the API for the image_widget_crop